operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -898,7 +898,9 @@ message ProxyConfig {
   TypeMapStringInterface lifecycle = 36;
 
   // Controls if sidecar is injected at the front of the container list and blocks the start of the other containers until the proxy is ready
-  google.protobuf.BoolValue holdApplicationUntilProxyStarts = 37;
+  //
+  // Deprecated: replaced by ProxyConfig setting which allows per-pod configuration of this behavior.
```

Comment:
can you add it here too https://github.com/istio/istio/blob/d13b044f471871116db3dedd5e4da995b2ca0096/operator/pkg/apis/istio/v1alpha1/validation/validation.go#L99? we don't expose deprecation directly from proto well